docs/scripts/formattedTSDemos.js

```diff
@@ -84,7 +84,8 @@ async function transpileFile(tsxPath, project) {
     const source = await fse.readFile(tsxPath, 'utf8');
 
     const transformOptions = { ...babelConfig, filename: tsxPath };
-    const enableJSXPreview = !tsxPath.includes(path.join('pages', 'premium-themes'));
+    const enableJSXPreview =
+      !tsxPath.includes(path.join('pages', 'premium-themes')) && !tsxPath.includes('templates');
```

Comment:
Just to be sure we do not block the creation of a `templates` folder for other teams

```suggestion
      !tsxPath.includes(path.join('pages', 'premium-themes')) && !tsxPath.includes(path.join('pages', 'templates'));
```

Reply:
Ah, I see in that case it should be 
```suggestion
      !tsxPath.includes(path.join('pages', 'premium-themes')) && !tsxPath.includes(path.join('getting-started', 'templates'));
```

I will update it 👍